app/assets/javascripts/discourse/widgets/notification-item.js.es6

```diff
@@ -148,6 +148,7 @@ createWidget("notification-item", {
     let infoKey = notName === "custom" ? data.message : notName;
     let text = emojiUnescape(this.text(notificationType, notName));
     let icon = iconNode(`notification.${infoKey}`);
+    let title = I18n.t(`notifications.titles.${notName ? notName : "custom"}`);
```

Comment:
I don't think we need the blank string to be translated. Something like the following should do

```
let title;

if (notName) {
  title = I18n.t(`notification.titles.${notName}`)
} else {
  title =  "";
}
```